packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -120,17 +112,19 @@ export default class QuanticSearchBox extends LightningElement {
 
   disconnectedCallback() {
     this.unsubscribe?.();
-    this.removeEventListener(
-      'suggestionlistrender',
-      this.handleSuggestionListEvent
-    );
+
+    this.removeEventListener('inputvaluechange', this.handleInputValueChange);
+    this.removeEventListener('submitsearch', this.handleSubmit);
+    this.removeEventListener('showsuggestions', this.handleShowSuggestions);
+    this.removeEventListener('selectsuggestion', this.handleSelectSuggestion);
   }
 
   updateState() {
     if (this.state?.value !== this.searchBox.state.value) {
-      this.input.value = this.searchBox.state.value;
+      // @ts-ignore
+      this.quanticSearchBoxInput.inputValue = this.searchBox.state.value;
```

Comment:
I thought we had fixed this type error no? You should be able to remove the `ts-ignore`